model/_init.cf

```diff
@@ -37,6 +37,11 @@ typedef ip_v10 as string matching ip::is_valid_ip_v10(self) == true
     A string type that matches both v4 and v6 cidraddresses (hence the v10)
 """
 
+typedef mask as string matching ip::is_valid_netmask(self)  == true
```

Comment:
In current versions of the language this should work as well.

```suggestion
typedef mask as string matching ip::is_valid_netmask(self)
```

Reply:
Fixed